packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -317,12 +431,10 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
           child: Semantics(
             button: true,
             child: ConstrainedBox(
-              constraints: widget.minSize == null
-                ? const BoxConstraints()
-                : BoxConstraints(
-                    minWidth: widget.minSize!,
-                    minHeight: widget.minSize!,
-                  ),
+              constraints: BoxConstraints(
+                minWidth: widget.minSize ?? kCupertinoButtonMinSize[widget.sizeStyle]!,
```

Comment:
I wonder if it'd be a good idea to change get these values from a static function using a switch statement, rather than a map, so that we don't have to use the `!` symbol?.